includes/admin.form.inc

```diff
@@ -16,7 +16,6 @@
  *   The Drupal form definition.
  */
 function islandora_entities_settings($form, &$form_state) {
-
```

Comment:
When doing a Pull Request that's the release branch version of a 7.x Pull request, it's gonna be a pain in the ass if it changes stuff that the other pull request doesn't.

Reply:
I know you've done a lot of work on this, but can we hold on for a moment? I'm not sure if this actually qualifies as a bug fix, and whether we can/should include it in the release. I'm hoping we can discuss this and #120 shortly. Thanks!